File: go/iq/spam~set~message+spam_list.go
// BuildSpamNewsletterReport builds <iq type="set" xmlns="spam">. Replies with one of: Success, Error.
func BuildSpamNewsletterReport(jid types.JID, spamFlow string, subject string, from types.JID) waBinary.Node {
	return waBinary.Node{
		Tag: "iq",
		Attrs: waBinary.Attrs{"type": "set", "to": "s.whatsapp.net", "xmlns": "spam"},
		Content: []waBinary.Node{
			{Tag: "spam_list", Attrs: waBinary.Attrs{"jid": jid, "spam_flow": spamFlow, "subject": subject}},
		},
	}
}

File: rust/iq/spam~set~message+spam_list.rs
/// <iq type="set" xmlns="spam">. Replies with one of: Success, Error.
pub fn build_spam_newsletter_report(jid: Jid, spamFlow: String, subject: String, from: Jid) -> Node {
    NodeBuilder::new("iq")
        .attr("type", "set")
        .attr("to", "s.whatsapp.net")
        .attr("xmlns", "spam")
        .child(NodeBuilder::new("spam_list").attr("jid", jid).attr("spam_flow", spamFlow).attr("subject", subject).build())
        .build()
}